src/rabbit_auth_backend_cache.erl

```diff
@@ -18,7 +18,6 @@
 -include_lib("rabbit_common/include/rabbit.hrl").
 
 -behaviour(rabbit_authn_backend).
--behaviour(rabbit_authz_backend).
```

Comment:
This plugin *is* supposed to provide an implementation of `rabbit_authz_backend`.